docs/install.rst

```diff
@@ -187,3 +340,45 @@ You can run the test suite as follows::
     lk.pytest()
 
 If all tests pass (except for the slow ones which are skipped) then your installation of `pylake` is good to go.
+
+
+**I tried the installation instructions but conda still won't install pylake**
+
+If creating a new environment doesn't work then it may be time to uninstall and reinstall conda.
+Note that this means you will lose all the environments you have created.
```

Comment:
This should be highlighted somehow, lest a user rushes to do the above step unaware of this effect.

Reply:
done